README.md

```diff
@@ -44,7 +44,7 @@ You can finetune Zoobot with a free GPU using this [Google Colab notebook](https
 If you will be making changes to the Zoobot package itself (e.g. to add a new architecture), download the code using git:
 
     # I recommend using a virtual environment, see below
-    git clone git@github.com:mwalmsley/zoobot.git
+    git clone https://github.com/mwalmsley/zoobot.git 
```

Comment:
Since this is part of the dev section, maybe we ought to suggest forking and `git clone`-ing that fork?

What do you think, @mwalmsley?